pkg/webhook/pvc/errors.go

```diff
@@ -20,24 +20,24 @@ import (
 	"fmt"
 )
 
-type validStorageClassError struct{}
+type storageClassNotValid struct{}
 
-func NewValidStorageClassError() error {
-	return &validStorageClassError{}
+func NewStorageClassNotValid() error {
+	return &storageClassNotValid{}
 }
 
-func (validStorageClassError) Error() string {
+func (storageClassNotValid) Error() string {
 	return "A valid Strage Class must be used"
```

Comment:
noticed a typo here: `Strage`, please can you fix it within your PR?

Reply:
fixed :)